web/api/netdata-swagger.json

```diff
@@ -839,6 +839,36 @@
         }
       }
     },
+    "/alarms_values": {
+      "get": {
+        "summary": "Get a list of active or raised alarms on the server",
+        "description": "The alarms_values endpoint returns the list of all raised or enabled alarms on the netdata server. Called without any parameters, all the enabled alarms are returned. By passing \"?active\",the raised alarms in state WARNING or CRITICAL are returned.",
+        "parameters": [
+          {
+            "name": "all",
+            "in": "query",
+            "description": "If passed, all enabled alarms are returned.",
+            "required": false,
+            "allowEmptyValue": true,
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An object containing general info and a linked list of alarms.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/alarms"
```

Comment:
Shouldn't there be a new one `alarms_values` ?

Reply:
I also fixed this, thank you very much!